src/platform/utilities/api/index.js

```diff
@@ -20,7 +20,9 @@ export function fetchAndUpdateSessionExpiration(...args) {
           localStorage.setItem('sessionExpiration', sessionExpiration);
         }
         // SSOe session is independent of vets-api, and must be kept alive for cross-session continuity
-        checkAndUpdateSSOeSession();
+        if (!environment.isProduction()) {
```

Comment:
why not in production?

Reply:
I'm just flagging it until we officially launch. 

This isn't connected to redux, so we'd either need to call the feature toggle endpoint directly to check if it's active (which maybe might be good for quickly shutting off anyway), or just wait until launching to remove this condition.

Reply:
I was able to wrap other calls to keep-alive with Flipper. However, looking at this again, `checkAndUpdateSSOeSession` returns if there's not an active SSO session, so this check is redundant and can be removed 🙈